app/src/main/java/com/dcrandroid/util/Utils.java

```diff
@@ -244,6 +247,13 @@ public static String removeTrailingZeros(double dcr){
         format.applyPattern("#,###,###,##0.########");
         return format.format(dcr);
     }
+    public static String formatToUsaStandard(long dcr) {
```

Comment:
What's formatToUsaStandard? It's a global app and not only for USA.

Reply:
> What's formatToUsaStandard? It's a global app and not only for USA.

renamed this method into "formatDecredWithComma"